app/javascript/mastodon/components/column_header.tsx

```diff
@@ -238,7 +239,10 @@ export const ColumnHeader: React.FC<Props> = ({
         onClick={handleToggleClick}
       >
         <i className='icon-with-badge'>
-          <Icon id='sliders' icon={SettingsIcon} />
+          <Icon
+            id='sliders'
+            icon={collapsed ? UnfoldMoreIcon : UnfoldLessIcon}
```

Comment:
I agree with the feedback someone else posted earlier that these icons imply an interaction where you'd select from a set of options, e.g. from Home to Explore.

Reply:
FWIW, the Twitter-like theme used on indieweb.social (forgot its name) uses a 90-degrees turned tune icon.

<img width="516" alt="image" src="https://github.com/user-attachments/assets/1fd12029-08d8-4aa6-b8f7-b1bf733fbc4b" />